metadata-models/src/main/pegasus/com/linkedin/mxe/DataHubUpgradeHistoryEvent.pdl

```diff
@@ -1,9 +1,9 @@
 namespace com.linkedin.mxe
 
 /**
- * Kafka event for recording a historical index rebuild change. Used for backwards incompatible changes to mappings and settings that require full reindexing.
+ * Kafka event for recording a historical version upgrade. Used for backwards incompatible changes to infrastructure that requires infrastructure level blocking changes.
  */
-record BuildIndicesHistoryEvent {
+record DataHubUpgradeHistoryEvent {
 
   /**
    * Version of the build
```

Comment:
Why is version of the build necessary? In the rest of upgrade fw we have a concept of a "version of an upgrade". Would be useful to do something similar here

Reply:
Forgot to change the doc here, yeah it can be version of upgrade instead.